### PR TITLE
New package: pdd-1.1.

### PR DESCRIPTION
**Pull request by maxice8**

### CODE REVIEW

File: srcpkgs/pdd/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+pkgname=pdd
+version=1.1
+revision=1
+build_style=gnu-makefile
+depends="python3-dateutil"
+# Tests aren't on 1.1 yet but they are already on upstream
+checkdepends="python3-dateutil python3-pytest"
+short_desc="Tiny date, time diff calculator"
+maintainer="maxice8 <thinkabit.ukim@gmail.com>"
+license="GPL-3.0-orlater"
```
Comment (newbluemoon) [inline srcpkgs/pdd/template]: Quick fingers again? ;)

**Comment by newbluemoon** [comment]
Looks like a neat tool. :)

File: srcpkgs/pdd/template (hunk 1 above)
Comment (maxice8) [reply srcpkgs/pdd/template]: yes

**Comment by jarun** [comment]
Any update on this?

**Comment by maxice8** [comment]
no, we just need to wait a Void Linux member merge this.

**Comment by maxice8** [comment]
if you want you can join #xbps on freenode and ask nicely for someone to merge it.

**Comment by jarun** [comment]
> if you want you can join #xbps on freenode and ask nicely for someone to merge it.

not happening ;)

**Pull request merged**